tests/unit/manage/test_forms.py

```diff
@@ -899,14 +899,12 @@ def test_validate_name_with_fewer_than_max_applications(self, db_session):
 
         form.validate__max_apps(pretend.stub())
 
-        assert form.errors == {
-            "__all__": [
-                (
-                    "You have already submitted the maximum number of "
-                    "Organization requests (3)."
-                )
-            ]
-        }
+        assert form.form_errors == [
+            (
+                "You have already submitted the maximum number of "
+                "Organization requests (3)."
+            )
+        ]
```

Comment:
This is the crux of the main API change - we could have kept iterating on `form.errors` but the key in their implementation is an empty string `""` instead of `"__all__"`, which we could customize the form class to override, but this felt cleaner.